schemas/examples/advanced/current.json

```
@@ -0,0 +1,17 @@
+{
+  "title": "examples/advanced",
+  "description": "A schema that references another schema",
+  "$id": "/examples/advanced/1.0.0",
+  "$comment": "status-draft",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "basic_from"
+  ],
+  "properties": {
+    "basic_from": {
+      "$ref": "/examples/basic/1.0.0"
+    }
+  }
+}
```

```suggestion
{
  "title": "examples/advanced",
  "description": "A schema that references another schema",
  "$id": "/examples/advanced/1.0.1",
  "$comment": "status-draft",
  "$schema": "http://json-schema.org/draft-07/schema#",
  "type": "object",
  "additionalProperties": false,
  "required": [
    "basic_from"
  ],
  "properties": {
    "basic_from": {
      "$ref": "/examples/basic/1.0.0"
    }, 
    "is_confidential": {
      "title": "Is Confidential",
      "type": "boolean",
      "description": "True if entry is confidential"
    }
  }
}
```

Adding a suggestion here to add a 1.0.1 version (then run a build), just so we have an example of updating a schema here.